console/src/main/scripts/plugins/metrics/ts/alertsCenterList.ts

```diff
@@ -57,16 +57,9 @@ module HawkularMetrics {
                 private $location:ng.ILocationService) {
       $scope.ac = this;
 
-      if ($routeParams.timeOffset === undefined) {
-        $routeParams.timeOffset = 3600000 * 12;
-        $scope.hkParams.timeOffset = $routeParams.timeOffset;
-      }
-
-      this.alertsTimeOffset = $routeParams.timeOffset;
-
       // If the end time is not specified in URL use current time as end time
       this.alertsTimeEnd = $routeParams.endTime ? $routeParams.endTime : Date.now();
-      this.alertsTimeStart = this.alertsTimeEnd - this.alertsTimeOffset;
+      this.alertsTimeStart = this.alertsTimeEnd - HawkularNav.getTimeOffset();
```

Comment:
@lucasponce, In case there's a timeOffset already set in the URL, that what we use.. if there isn't any, we use the default (now 12h) as in https://github.com/hawkular/hawkular/blob/6eb079e181ec7cfd98974c57bde65a85b3fb53a6/console/src/main/scripts/plugins/directives/topbar/ts/topbarServices.ts#L47L58.

We should rely on hkParams.timeOffset and always provide our links making use of it. We are losing the range selection when moving between top level pages, eg, the Alerts Center link at https://github.com/hawkular/hawkular/blob/6eb079e181ec7cfd98974c57bde65a85b3fb53a6/console/src/main/scripts/plugins/directives/topbar/html/topbar.html#L3 should be `<a href="/hawkular-ui/alerts-center/{{hkParams.timeOffset}}">Alert Center</a>`